helper/token.js

```diff
@@ -0,0 +1,79 @@
+const promise = require('bluebird');
+const dbHelper = require('../helper/database');
+const request = require('request');
+const tokenCollection = 'token';
+const canvasUrl = 'http://canvas.differ.chat/api/v1/users/36/courses?access_token=';
```

Comment:
please use 'self' instead of hardcoding the user-id. And this URL should be in the config-file as well :)